installers/windows/minikube.nsi

```diff
@@ -0,0 +1,131 @@
+# This installs two files, minikube.exe and logo.ico, creates a start menu shortcut, builds an uninstaller, and
```

Comment:
More of a nit than anything.  NSIS functions are capitalized, 'Function', 'Section', etc.  They don't have an official style guide but this seems to be the standard.